src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  // знаходжу основений клас(обгортку)
+  const gallery = document.querySelector('#thumbs');
+  const mainImage = document.querySelector('#largeImg');
+
+  gallery.addEventListener('click', (e) => {
+    e.preventDefault();
```

Comment:
Consider moving `e.preventDefault();` inside the `if` block on line 14. This way, the default action is only prevented when the click is on a thumbnail image, which is likely the intended behavior. Currently, it prevents the default action for all clicks within the gallery, which might not be necessary.